providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/resolver/process/targeting/OperatorTest.java

```diff
@@ -18,6 +19,20 @@ static void setUp() {
         OPERATOR = new Operator();
     }
 
+    @Test
+    void flagKeyPresent() throws TargetingRuleException {
```

Comment:
new test asserts that `$flagd.flagKey` === the flag key

Reply:
Nice !  this is a  really cool test 👍